src/System.Net.Sockets/src/System/Net/Sockets/UDPClient.cs

```diff
@@ -233,11 +233,7 @@ protected virtual void Dispose(bool disposing)
         {
             if (disposing)
             {
-                if (GlobalLog.IsEnabled)
-                {
-                    GlobalLog.Print("UdpClient::Dispose()");
-                }
-
+                NetEventSource.Info(this);
```

Comment:
There wasn't in the original, so I didn't add any.  The original only traced out the name of the caller, which happens implicitly in this revised version.